### ISSUE TITLE
Darkmode Request

### ISSUE THREAD

**Issue by SoundSherpa** [eXaminator/kanka-foundry#79]
Hello! This is obviously low priority, but is it possible to request a darkmode for the journal sheets?
Thank you.

**Comment by eXaminator** [comment]
Hi @SoundSherpa, thanks for the request.

Though, I'm not quite sure this is something I'd like to get into. The reasoning is, that this module fits the default foundry styles. I have no control over what kind of theming modules users install. Just adding a "dark mode" might interfere with other themes.

Are you using some specific foundry theme module? I'd like to look into that and see if theres anything I can do to make my module play better with theming modules in general. But since there is no standard of how to do theming, you'll probably always have trouble with certain modules that add new UI.

**Comment by eXaminator** [comment]
The whole sheet selection is something that is pretty new (and only via mod, so not officially supported yet) for journals. Generally speaking, I would not count on the sheet selection always being available in the future when it comes to kanka sheets (as it does not make sense to change the sheets to anything but the kanka sheet - just as it doesn't make sense to use the kanka sheet for other entries - it simply won't work as expected).

That being said, my problem with this is, that there are basically 2 modules (3 if I were to follow suite) that provide their own dark themes, which might look nothing alike. Sadly, I don't think foundries UI implementation is robust (or extensive) enough to ensure that global theme modules can actually style everything (this includes other modules) as expected.

TLDR: I'm not sure what the best course of action or general solution is for this. I'll ponder this question for a while, maybe I'll bring it up with some other devs in this space, and see what people think makes most sense. I understand your need and hopefully we'll find a solution for it, but probably not in the very near future.

That being said, a simple workaround could be to use a module like https://foundryvtt.com/packages/custom-css and add custom styles that change the appearance of the kanka sheet - but this requires some knowledge of CSS (so it's maybe not THAT easy if you'd have to learn it first...).

**Comment by SoundSherpa** [comment]
I absolutely understand. Thanks!

**Comment by Nedrapter** [comment]
I'm also using Kanka in dark mode, so it's not looking very nice in the bright foundry theme. I wouldn't want to change the whole foundry theme, so if something can be done for the module's entries only, it'd be nice!

**Comment by eXaminator** [comment]
I will close this issue now. I'm not planning to provide different themes for this module in the foreseeable future. The upcoming update for this module will completely overhaul the kanka sheets so that they use (mostly) default foundry styles and look/work mostly like regular foundry journal sheets.

This means that any module that changes styles for normal journal entries should also work pretty well with this module. If you want different the colors schemes for these sheets you could always use a module that allows you to define custom CSS for your world and change the colors yourself. I'm sure you'll find people in the Discord that can help you. Anything that changes colors on a normal journal sheet should also work for these new sheets. Or you can apply styles using the `kanka-journal` css class if you just want to change the kanka journals.
